broker-cli/commands/health-check.js

```diff
@@ -58,16 +58,25 @@ const ENGINE_STATUS_CODES = Object.freeze({
  */
 
 async function healthCheck (args, opts, logger) {
-  const { rpcAddress } = opts
+  const {
+    rpcAddress,
+    json
+  } = opts
 
   try {
     const client = new BrokerDaemonClient(rpcAddress)
 
+    const res = await client.adminService.healthCheck({})
+
+    if (json) {
+      return console.log(JSON.stringify(res))
```

Comment:
Not sure about this: do we need to use `logger` for this to show up in scalyr?

Also, low priority, in `wallet.js` we do a pretty print (`JSON.stringify(transactions, null, 2`). Should we create a ticket to remove the pretty printing so it can also be consumed by other programs more easily?

Reply:
@bradleystachurski this is such a good observation. Thanks for the catch.

1. we should use logger, i will make the change
2. pretty printing is actually what we want to do, i've added this to the code

Pretty printing is only there to see it visually (the same for transactions). If there are any use-cases where this doesn't allow someone to use the `--json` flag effectively, then we can modify.

For us, our current use would be to make sure that different keys in the response are present and I wanted to do this through JSON instead of having to parse the stdout table (although they are arguably the same)

As an example, from the way the code is now, we can see if relayer has RELAYER_OK by running the following:

```
sparkswap healthcheck --json | grep 'RELAYER_OK' | wc -l
```

Returning 1 if we are OK and returning 0 if something is wrong

Reply:
Gotcha, that makes sense.